data/schema-ext.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Resume Schema Extension for Test Double",
+  "type": "object",
+  "properties": {
+    "skills": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "level": {
+            "type": "string",
+            "enum": ["Teaching others", "Pairing on", "Excited about"]
```

Comment:
Learned something about JSON schemas, very cool that they can be extended this way.

I like the intention behind this, making clear that we don't find much value in using "level" to rank our skills/languages etc. These specific categories don't resonate with me though, so I am glad for now that validation doesn't stop the build ;-) More specifically, they actually apply very well to all the skills I want to list on my resume, I am excited about and love to pair and teach (and learn) all of them!

I have some ideas I would like to mess with, but I'll express them in the form of a PR once I get some time to work on it.